### DISCUSSION TITLE
How to correctly setup Configuration File to config JSON file formatting?

### DISCUSSION THREAD

**Discussion by Ted-Rose** [prettier/prettier#15258] [Q&A]
Here's the corrected version:

My goal is to set up Prettier to use tabs for indentation instead of spaces for JSON files using Configuration File.

So far, I have done the following, but when formatting JSON files in VS Code with Prettier, indentation is still done using spaces instead of tabs:

1. Install Node.js:
   ```
   brew install node
   ```

2. Install cosmiconfig for configuration file support:
   ```
   npm install cosmiconfig
   ```

3. Add a `.prettierrc.yaml` file within the project's file tree with this content:
trailingComma: 'es5'
tabWidth: 4
indent_style: 'tab'
semi: false
singleQuote: true
insert_final_newline: false

Any advice would be appreciated.

**Comment by seiyab** [comment]
Are you using Prettier plugin for VSCode?
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

AFAK, we don't need to install cosmiconfig though Prettier uses it internally.

**Reply by Ted-Rose** [reply]
Yes, I have installed Prettier

**Reply by seiyab** [reply]
Sorry for late response. What happen when you Cmd + Shift + P -> Format Document With... -> Prettier?
<img width="643" alt="image" src="https://github.com/prettier/prettier/assets/20365512/b4a0fe18-a325-45aa-abc1-e513a22ed312">
